frontend/src/component/releases/ReleasePlanTemplate/TemplateForm/MilestoneList/MilestoneCard/MilestoneCard.tsx

```diff
@@ -106,25 +115,34 @@ const StyledAccordionFooter = styled('div')(({ theme }) => ({
     justifyContent: 'flex-end',
     gap: theme.spacing(3),
     backgroundColor: 'inherit',
-    borderRadius: theme.shape.borderRadiusMedium,
 }));
 
 const StyledIconButton = styled(IconButton)(({ theme }) => ({
     marginLeft: theme.spacing(1),
     color: theme.palette.primary.main,
 }));
 
-const StyledDragIcon = styled(IconButton)(({ theme }) => ({
+const DragButton = styled('button')(({ theme }) => ({
```

Comment:
We override all the MUI stuff anyway, so might as well use a regular ol' button.